crates/astria-sequencer/src/asset/mod.rs

```diff
@@ -0,0 +1,16 @@
+use once_cell::sync::OnceCell;
+use proto::native::sequencer::v1alpha1::asset::Denom;
+
+pub(crate) static NATIVE_ASSET: OnceCell<Denom> = OnceCell::new();
```

Comment:
`once_cell::sync::Lazy` might be a slightly nicer API for this.

Reply:
don't think we can use `Lazy` as it requires an initializer function but we need to set this at runtime (ie. either from value in storage, or in `init_chain`)